install.sh

```diff
@@ -59,13 +61,42 @@ function install_apt {
     fi
 }
 
+function install_yum {
+    local CMD=""
+    if [ -n "${YUM_DNF}" ]; then
+        CMD=$YUM_DNF
+    else
+        if [ -n "${YUM_YUM}" ]; then
+            CMD=$YUM_YUM
+	fi
+    fi
+
+    if [ "${CMD}" != "" ]; then
+        local PARAMS="--assumeyes"
```

Comment:
Did you test this on Ubuntu? From a quick google it looks like `--assume-yes` (with the `-` separator) is the param for `apt-get`. Does `--assumeyes` work OK?